tools/backups/db-backup/db-backup.dc.json

```diff
@@ -0,0 +1,590 @@
+                  "requests": {
+                    "cpu": "${CPU_REQUEST}",
+                    "memory": "${MEMORY_REQUEST}"
+                  },
+                  "limits": {
+                    "cpu": "${CPU_LIMIT}",
+                    "memory": "${MEMORY_LIMIT}"
+                  }
+                },
+                "volumeMounts": [
+                  {
+                    "name": "${BACKUP_VOLUME_NAME}",
+                    "mountPath": "${BACKUP_DIR}"
+                  },
+                  {
+                    "name": "${VERIFICATION_VOLUME_NAME}",
+                    "mountPath": "${VERIFICATION_VOLUME_MOUNT_PATH}"
+                  },
+                  {
+                    "name": "${NAME}-config-volume",
+                    "mountPath": "${CONFIG_MOUNT_PATH}${CONFIG_FILE_NAME}",
+                    "subPath": "${CONFIG_FILE_NAME}"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the resources.  Use 'backup-{database name}' depending on your database provider",
+      "required": true,
+      "value": "backup-mongodb"
+    },
+    {
+      "name": "SOURCE_IMAGE_NAME",
+      "displayName": "Source Image Name",
+      "description": "The name of the image to use for this resource.  Use 'backup-{database name}' depending on your database provider",
+      "required": true,
+      "value": "backup-mongodb"
+    },
+    {
+      "name": "IMAGE_NAMESPACE",
+      "displayName": "Image Namespace",
+      "description": "The namespace of the OpenShift project containing the imagestream for the application.",
+      "required": true,
+      "value": "iarjfo-tools"
+    },
+    {
+      "name": "TAG_NAME",
+      "displayName": "Environment TAG name",
+      "description": "The TAG name for this environment, e.g., dev, test, prod",
+      "required": true,
+      "value": "prod"
+    },
+    {
+      "name": "DATABASE_SERVICE_NAME",
+      "displayName": "Database Service Name",
+      "description": "Used for backward compatibility only. Not needed when using the recommended 'backup.conf' configuration.  The name of the database service.",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "DATABASE_NAME",
+      "displayName": "Database Name",
+      "description": "Used for backward compatibility only. Not needed when using the recommended 'backup.conf' configuration.  The name of the database.",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "MONGODB_AUTHENTICATION_DATABASE",
+      "displayName": "MongoDB Authentication Database",
+      "description": "This is only required if you are backing up mongo database with a separate authentication database.",
+      "required": false,
+      "value": "admin"
+    },
+    {
+      "name": "POSTGRES_DEPLOYMENT_NAME",
```

Comment:
Do we need the other configurations for Postgres or MSSQL in the json still? Doesn't look like they're getting used.

Reply:
Good point, I was just using the provided templates. I tried removing them however and the deployment just hangs in a pending state. The BCDevOps backup script is probably still looking for them at some level unfortunately.